fvm/evm/testutils/contracts/test.sol

```diff
@@ -48,6 +48,13 @@ contract Storage {
         selfdestruct(payable(msg.sender));
     }
 
+    function verifyArchCallToRandomSource(uint64 height) public view returns (uint64) {
+        (bool ok, bytes memory data) = cadenceArch.staticcall(abi.encodeWithSignature("getRandomSource(uint64)", height));
+        require(ok, "unsuccessful call to arch ");
+        uint64 output = abi.decode(data, (uint64));
+        return output;
```

Comment:
I usually use the verify methods to verify the output matches what is expected, for example see the verifyArchCallToFlowBlockHeight , we check the output produced matches the expected.

Reply:
I did that in Go test, but yeah might be ok to adpot the pattern you made